docs/sphinx/source/user_guide/glossary.rst

```diff
@@ -0,0 +1,192 @@
+.. glossary:
+
+Glossary
+========
+
+There is a convention on consistent variable names throughout the library:
+
+.. glossary::
+
+    airmass
+        Airmass
+    
+    airmass_absolute
+        Absolute airmass
+    
+    airmass_relative
+        Relative airmass
+    
+    albedo
+        Ratio of reflected solar irradiance to global horizontal irradiance
+        [unitless]
+    
+    aoi
+        Angle of incidence. Angle between the surface normal vector and the
+        vector pointing towards the sun’s center
+    
+    aoi_projection
+        cos(aoi)
+
+    ape
+        Average photon energy
+
+    apparent_zenith
+        Refraction-corrected solar zenith angle in degrees
+
+    bhi
+        Beam/direct horizontal irradiance
+
+    dhi
+        Diffuse horizontal irradiance
+
+    dni
+        Direct normal irradiance [Wm⁻²]. Irradiance received per unit area by a
+        surface perpendicular (normal) to the sun's rays that propagate in a
+        straight line from the sun.
```

Comment:
Is there a reason why DNI has an extensive explanation of what it is (including units) while the other irradiances don't? I kind of like the idea of reporting the unit that each term is (usually) measured in pvlib.